### ISSUE TITLE
Release version 8.2.0

### ISSUE THREAD

**Issue by sirosen** [marshmallow-code/webargs#741]
I meant to do this _a while ago_, and just realized that I never circled back to do it.

@lafrech, I'm mostly creating this issue for us to sync / confirm that we're happy to release this with the current code in `dev`.

I may be able to put some energy today and tomorrow into an example gallery for the docs (I've only been punting on that for 6 months! 😅 ), so it may make sense to wait a couple of days. I was planning to release 8.2.0 in the middle of the coming week, to give a couple of days to see if I can get that working.

**Comment by lafrech** [comment]
Sure.

I totally forgot about it when doing my periodic review of open issues/PRs in marshmallow libs.

Yeah, do what you fell appropriate. You may ship as is, or add examples to the docs if you wish.

Thanks.

**Comment by sirosen** [comment]
As it turns out, I had a lot of trouble in the past week making extra time for this.

I just did the push to roll out the release, and it looks like the build failed ( https://github.com/marshmallow-code/webargs/runs/7285388668?check_suite_focus=true ):
![image](https://user-images.githubusercontent.com/1300022/178301778-d32877f3-486a-4ad7-894e-568c7d2b9a46.png)

I'm not sure how `lint-pre-release` differs from the other linting, but I'm digging into this. Once I get it sorted out, I'm not sure what the best steps to take to "re-release" 8.2.0 are.

I _think_ the appropriate thing to do is to delete the `8.2.0` tag and recreate it? Let me know if that seems good or bad. I'll wait before taking action on that.

**Comment by lafrech** [comment]
I wouldn't delete a published tag.

When there is an issue with a released package, the way to go, I believe, is to remove the package and publish a fixed release with a different name. This is was Steven did with 4.0.0b1.post0 in [apispec](https://pypi.org/project/apispec/#history) after I screwed up 4.0.0b1 release.

Here, the case is a bit different. The package was not released. Besides, the error is not even in the published packaged, only with meta stuff (CI) not uploaded to PyPI, so I don't mind forcing the release from the same tag, which I just did: https://pypi.org/project/webargs/.
